index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<link href="registration.css" rel="stylesheet" type="text/css"> </head>
+
+<body>
+	<form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" name="test-form">
+		<fieldset class="personalInfo">
+			<legend>Personal information</legend>
+			<div class="form-field"> Surname:
+				<input type="text" name="surname" required> </div>
+			<div class="form-field"> Name:
+				<input type="text" name="name" required> </div>
+			<div class="form-field"> How old are you?
+				<input type="number" name="age"  minlength="1" maxlength="100" placeholder="12" required> </div>
+			<div class="form-field"> Full date of birth?
+				<input type="date" name="birthdate" placeholder="mm/dd/yyyy" required> </div>
+			<div class="form-field"> I accept the terms of the agreement
+				<input type="checkbox" name="terms" id="terms-and-conditions" required>
+				<label for="terms-and-conditions"></label>
+			</div>
+		</fieldset>
+		<fieldset class="registration">
+			<legend>Registration</legend>
+			<div class="form-field"> E-mail:
+				<input type="email" name="email" placeholder="email@example.com" required> </div>
+			<div class="form-field"> Password:
+				<input type="password" name="password"  autocomplete="off" required> </div>
+		</fieldset>
+		<fieldset class="interestingFact">
+			<legend>An interesting fact about you!</legend>
+			<div class="form-field" required> Do you love cats?
+				<label>
+					<input type="radio" name="animal" value="yes" required> Yes </label>
+				<label>
+					<input type="radio" name="animal" value="no"> No </label>
+			</div>
+			<div class="form-field"> What is your favorite color?
+				<input type="color" name="favorite-color" placeholder="#e0000" required> </div>
+			<div class="form-field"> What time do you go to bed?
+				<input type="time" name="bedtime" value="--:--:--" required> </div>
+			<div class="form-field"> What are your favorite brands of cars?
+				<select name="auto" multiple required>
+					<option value="BMW">BMW</option>
+					<option value="Audi">Audi</option>
+					<option value="Lada">Lada</option>
+				</select>
+			</div>
+			<div class="form-field"> How do you rate our work?
+				<label>
+					<input type="range" name="rating" min="0" max="11" required> </label>
+			</div>
+		</fieldset>
+		<fieldset class="additionalInfo">
+			<legend>Additional info</legend>
+			<div>
+				<label class="form-field"> Comments:
+					<textarea name="feedback" id="" autocomplete="off"></textarea>
```

Comment:
```suggestion
					<textarea
					  name="feedback"
					  autocomplete="off"
			 ></textarea>
```

Remove attribute id with empty value and format textarea due to the [checklist ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)to avoid extra spaces in textarea field

![image](https://user-images.githubusercontent.com/80700247/226168620-2d8ce3e6-97f9-4943-bc3b-3608fe9029ca.png)